app/javascript/Redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from "@reduxjs/toolkit";
```

Comment:
- [ ] It seems like you used the `@reduxjs/toolkit` as per the project requirements, but while generating the build file, I found that you haven't added the `@reduxjs/toolkit` as the dependency for the project, and because of it I got the build error for the library. Kindly add the library in your `package.json`, so that it would be convenient for any person to use your project without any issue.

![image](https://github.com/ismayito/hello-rails-react/assets/66476329/58a4c20f-22bb-45c8-bc05-35cf0db1c447)